setup.cfg

```diff
@@ -0,0 +1,41 @@
+[flake8]
+ignore = N802,W503,E402
+max-line-length = 160
+exclude = migrations,.ropeproject,static,_static,build
+
+[isort]
+combine_as_imports = true
+default_section = THIRDPARTY
+include_trailing_comma = true
+known_third_party = pretix
+known_standard_library = typing
+multi_line_output = 5
+not_skip = __init__.py
+skip = setup.py
+
+[tool:pytest]
+DJANGO_SETTINGS_MODULE = pretix.testutils.settings
+
+[coverage:run]
+source = exhibitors
+omit = */migrations/*,*/urls.py,*/tests/*
+
+[coverage:report]
+exclude_lines =
+    pragma: no cover
+    def __str__
+    der __repr__
+    if settings.DEBUG
+    NOQA
+    NotImplementedError
+
+[check-manifest]
+ignore =
+    .update-locales
+    .update-locales.sh
+    .gitlab-ci.yml
```

Comment:
delete it please and check other parts of setup.cfg

Reply:
Removed 👍